server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -0,0 +1,182 @@
+Feature: GetMessages method
+  As a James user
+  I want to be able to retrieve my messages
+
+  Background:
+    Given a domain named "domain.tld"
+    And a current user with username "username@domain.tld" and password "secret"
+    And the current user has a mailbox named "inbox"
+
+  Scenario: Retrieving messages with a non null accountId should return a NotSupported error
+    When the user is getting his messages and gives a non null accountId
```

Comment:
Ok, but without "inbox" as fetching in a dedicated mailbox is not in the GetMessages specification.